stories/Autocomplete.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, boolean, withKnobs } from '@storybook/addon-knobs';
+import { Autocomplete, IconMatch } from '@textkernel/oneui';
+import {
+    SUGGESTIONS,
+    SUGGESTION_TO_STRING
+} from '../src/components/Autocomplete/__mocks__/suggestions';
+
+storiesOf('Autocomplete', module)
+    .addDecorator(withKnobs)
+    .add('Autocomplete as single select with icon', () => (
```

Comment:
With controlled components, it's really hard to feel how they work in real when you put them in the soprybook as is (without a wrapper that actually controls them).
I think it should be a requirement for controlled components to be "storybooked" with such kind of wrappers.

Reply:
I'm not sure what you mean. Can you give an example

Reply:
I mean that if you'd open our storybook, and try use Autosuggest (or some other components), it feels like it doesn't really work. Because it's a controlled component and it has no wrapper that really controls it, only static props. So you can not select/delete/etc anything.
So it's hard to evaluate the component from UX perspective.